specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/networkVerifier.json

```diff
@@ -0,0 +1,1247 @@
+        "responses": {
+          "200": {
+            "description": "Get a reachabilityAnalysisRun",
+            "schema": {
+              "$ref": "#/definitions/ReachabilityAnalysisRun"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CommonErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ReachabilityAnalysisRunGet": {
+            "$ref": "./examples/ReachabilityAnalysisRunGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ReachabilityAnalysisRuns"
+        ],
+        "summary": "Creates Reachability Analysis Runs.",
+        "operationId": "ReachabilityAnalysisRuns_Create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "workspaceName",
+            "description": "Workspace name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_.-]*$"
+          },
+          {
+            "in": "path",
+            "name": "reachabilityAnalysisRunName",
+            "description": "Reachability Analysis Run name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_.-]*$"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Analysis Run resource object to create/update.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReachabilityAnalysisRun"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ReachabilityAnalysisRun updated.",
+            "schema": {
+              "$ref": "#/definitions/ReachabilityAnalysisRun"
+            }
+          },
+          "201": {
+            "description": "ReachabilityAnalysisRun created and will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/ReachabilityAnalysisRun"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CommonErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
```

Comment:
>"x-ms-long-running-operation": true, [](http://example.com/codeflow?start=8&length=36)

same here 

"x-ms-long-running-operation-options": {
                    "final-state-via": "location"
                }

Reply:
same as here https://github.com/Azure/azure-rest-api-specs/pull/30560/files#r1764369061

Reply:
this is a new change not reviewed